pkg/query.go

```diff
@@ -12,17 +15,41 @@ var DefaultQuery = "SELECT 1 FORMAT JSON;"
 var TimeZoneFieldName = "timezone()"
 var TimeZoneQuery = fmt.Sprintf("SELECT %s FORMAT JSON;", TimeZoneFieldName)
 
+/* TODO Ugly hack cover 80% corner cases, think about how to port sql_query.ts+scanner.ts to Golang, or try to figure out howto tricksterproxy.io parse SQL query to detect timeRange */
```

Comment:
@Slach additionally, the backend can consume a url, such that instead of building a frontend and backend query, the frontend could forward the query into the backend to handle all of the query processing and return a frontend friendly result.

Reply:
we change SQL queries too often in query editor, I think it will request storm

only one thing we need is detect to where is $from and $to macros contains in `formattedQuery` present in `rawQuery` and try to replace it for actual values

or try parse `rawQuery` to detect which tokens are timestamps and replace it